### DISCUSSION TITLE
"Open in New Tab" notification alert

### DISCUSSION THREAD

**Discussion by Mircobrb** [zen-browser/desktop#2248] [Q&A]
I was wondering is there a setting to enable a notification when you open a link in a new tab? Like here in Arc Browser

![image](https://github.com/user-attachments/assets/0e575405-2b22-4fbc-887e-48e95c6efb4a)

**Comment by espalloc7** [comment]
would be great. i support

**Comment by AksharDP** [comment]
+1

**Comment by Paul2389** [comment]
That would be a great addition.
